app/views/people/_form_basics.haml

```diff
@@ -19,6 +19,9 @@
   = form.label :suffix
   = form.select :suffix, [nil]+Setting.get(:system, :suffixes), {}, class: 'form-control'
 .form-group
+  = form.label :alias
```

Comment:
Can you add `alias` to here please: https://github.com/churchio/onebody/blob/master/config/locales/en/general.yml#L100